packages/doenetml-worker-rust/lib-doenetml-core/src/components/doenet/number.rs

```diff
@@ -0,0 +1,66 @@
+use crate::components::prelude::*;
+use crate::general_prop::NumberProp;
+use crate::general_prop::NumberToStringProp;
+use crate::props::UpdaterObject;
+
+/// The `<number>` component calculates a numerical value from its contents and display the result as text
```

Comment:
Does it display fractions as well as decimals?

Reply:
No, it is an `f64` representation, so only decimals. This is approximately equivalent to the legacy `<number>` component, which uses a Javascript `Number`. At this point, if you want fractions, you have to stick with `<math>`.